scripts/lazygit_auto_commit.sh

```
@@ -0,0 +1,273 @@
+    log_message "=== コミットウィンドウ表示失敗で終了 ==="
+    log_message "代替手順: 手動で'c'キーを押してコミット画面を開いてください"
+    log_message "生成されたコミットメッセージは既にテンプレートに設定済みです"
+fi
+
+# コミット完了後のクリーンアップ処理
+log_message "=== ポストコミットクリーンアップ開始 ==="
+
+# 成功時のみクリーンアップを実行
+if [[ "$success" == "true" ]]; then
+    # 設定可能な待機時間（環境変数で設定可能、デフォルト500ms）
+    local cleanup_delay_ms="${AI_COMMIT_CLEANUP_DELAY_MS:-500}"
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)